tests/test_data_preparation.py

```diff
@@ -24,25 +34,50 @@ def compute_md5(file_path):
     return hash_md5.hexdigest()
 
 
+def docker_rm_and_mkdir(file_):
```

Comment:
why not take the dir here? Feels a bit more natural as you're removing the full parent folder anyway, so I'd just pass it in

Reply:
Not exactly, we're deleting the file and making sure the directory we want to put it into exists, but not deleting the other contents of that directory. Happy to do it otherwise, but I'm not sure it's worth our time to vex over.

Reply:
ah sorry - missed that. Let's keep the current way